controller/src/beerocks/master/db/node.h

```diff
@@ -182,6 +186,26 @@ class node {
     };
     std::shared_ptr<radio> hostap;
 
+    class link_metrics_data{
+        public:
+        
+        link_metrics_data(){};
+        ~link_metrics_data(){};
+        
+        sMacAddr al_mac_of_the_device_that_is_reported;
+        std::list<ieee1905_1::tlvTransmitterLinkMetric::sInterfacePairInfo> transmitterLinkMetrics;
```

Comment:
Usually the list of interface pairs will contain just one element. Also, we never add or remove from the list, when we receive a linkMetricsTLV we always get the complete list. So a vector is more appropriate.